pac4j-jwt/src/main/java/org/pac4j/jwt/profile/JwtGenerator.java

```diff
@@ -38,27 +46,28 @@
 
     protected final Logger logger = LoggerFactory.getLogger(getClass());
 
-    private final String secret;
-    private boolean encrypted = true;
+    private final String signingSecret;
+    private final String encryptionSecret;
 
     /**
-     * Initializes the generator that will create JWT tokens that are both signed and encrypted.
+     * Initializes the generator that will create JWT tokens that are signed but not encrypted.
      *
-     * @param secret The secret. Must be at least 256 bits long and not {@code null}
+     * @param signingSecret The signingSecret. Must be at least 256 bits long and not {@code null}
      */
-    public JwtGenerator(String secret) {
-        this.secret = secret;
+    public JwtGenerator(final String signingSecret) {
+        this(signingSecret, null);
```

Comment:
I guess there is a problem with the `null`: it should be `signingSecret`, right?